PowerShell/BuildingBlocks/CustomerAndResources/CreateCustomProperty.ps1

```diff
@@ -0,0 +1,28 @@
+﻿param(
+  [Parameter(Mandatory=$true)]
+  [string]$Name,
+  [Parameter(Mandatory=$true)]
+  [string]$Value,
+  [Parameter(Mandatory=$true)]
+  [string]$ReferenceEntityType,
+  [Parameter(Mandatory=$true)]
+  [string]$ReferenceEntityId)
+
+# Initialize the session
+.\Init.ps1
+
+# Check if a custom property with the same fields already exists
+# Name: The name of the custom property.
+# Value: The value of the custom property.
+# ReferenceEntityType: The type of the entity associated with the custom property. Examples include 'Customer', 'CustomerDevice', 'CustomerEndUser'.
+# ReferenceEntityId: The id of the entity associated with the custom property.
+$existingCustomProperty = Get-BT_ExtendedProperty -Ticket $mspc.WorkgroupTicket -Name $name -ReferenceEntityType $ReferenceEntityType -ReferenceEntityId $ReferenceEntityId
+if ( $existingCustomProperty -ne $null ) {
```

Comment:
Remove spaces inside ( and ) for consistency.

Reply:
Fixed